tests/integration/cli/test_console.py

```diff
@@ -4,13 +4,99 @@
 from tests.integration.cli.utils import skip_projects
 
 
+def no_console_error(result):
+    return "NameError" not in result.output and "AssertionError" not in result.output
+
+
+def destroy_ape_console_extras(project):
+    try:
+        project.config_manager.DATA_FOLDER.joinpath("ape_console_extras.py").unlink()
+    except FileNotFoundError:
```

Comment:
You could jut check `is_file()` first rather than doing a whole try and catch (twice)

Reply:
Works for me